javascript/modules/client/src/client.ts

```diff
@@ -91,12 +93,16 @@ export class Client extends ClientCore implements IClient {
     /** Retrieves the list of asset transfers to and from the given DAO, by default, from ETH, DAI, USDC and USDT on Mainnet*/
     getTransfers: (daoAddressOrEns: string) =>
       this._getTransfers(daoAddressOrEns),
+    /** Retrieves the list of plugin installed in the DAO*/
+    getInstalledPlugins: (daoAddressOrEns: string): Promise<string[]> =>
+      this._getInstalledPlugins(daoAddressOrEns),
     /** Retrieves metadata for DAO with given identifier (address or ens domain)*/
     getMetadata: (daoAddressOrEns: string) =>
       this._getMetadata(daoAddressOrEns),
-    /** Retrieves list of created DAOs and the corresponding metadata*/
-    getMetadataMany: (options?: DaoQueryOptions) =>
-      this._getMetadataMany(options),
+    /** Retrieves metadata for DAO with given identifier (address or ens domain)*/
+    getMetadataMany: (params?: IDaoQueryParams): Promise<DaoMetadata[]> =>
+      this._getMetadataMany(params ?? {}),
```

Comment:
As suggested earlier, from the consumer perspective, it could have more sense to provide `getDao(...)` and `getDaos(...)` as opposed to only fetching the metadata. Names would sound more natural, but then we would be fetching metadata + pluginList + ...

Thoughts on this?